src/main/java/com/powsybl/openloadflow/equations/Equation.java

```diff
@@ -104,12 +102,22 @@ private static double getBusTargetV(LfBus bus) {
         }
     }
 
-    private static double getBranchPhi(LfBranch branch) {
+    private static double getBranchA(LfBranch branch) {
```

Comment:
Here ````A```` is the same A that we found in ````deriveA```` ? In fact you choose ````deriveA```` because you compute the derivative of (A2 - A1) but A2 is fixed ?

Reply:
I will change that allow both a1 and a2 phase shifting everywhere in equation system even if not used right now when connected to IIDM model. Here because of a2=0 hardcoded we confuse a1 - a2 and a1.